atlasdb-api/build.gradle

```diff
@@ -4,6 +4,8 @@ apply plugin: 'com.palantir.revapi'
 dependencies {
   api project(":lock-api")
   api project(":timestamp-api")
+  api 'com.palantir.conjure.java.api:service-config'
```

Comment:
Might be missing something: I'm guessing this is required because of dialogue clients?

Reply:
ah, sorry, I'll remove. I initially also plumbed UserAgents, until I realized that's redundant

Reply:
Dne